README-Client.txt

```diff
@@ -53,16 +53,19 @@ hubmap_api_py_client.Client = class ExternalClient(builtins.object)
  |      Returns:
  |          ResultsSet
  |  
- |  select_datasets lambda self, where=None, has=None
+ |  select_datasets lambda self, where=None, has=None, genomic_modality=None, logical_operator=None, min_cell_percentage=None
  |      Select a set of datasets. If no params are provided, selects the set of all datasets.
  |      Otherwise, selects a set of datasets filtered based on parameters supplied.
  |      
  |      Args:
  |          where (str): The type of entity for which identifiers are supplied as input to query.
  |              Must be one of ["dataset", "cell", "cluster"].
- |          has (List[str]): A list of entity identifiers (dataset_uuids, etc) supplied as input to
- |              the query.
- |      
+ |          has (List[str]): A list of entity identifiers or expressions (dataset_uuids, etc) supplied as
```

Comment:
`expressions` are the inequalities? It would be good to document them somewhere... So far, I think this regex would capture what I've seen:
```
\w+ (<|>) \d+(\.\d+)?
```
but maybe you support having a gene symbol on both sides? Or maybe you support booleans? I'm not arguing to expand the feature set, but I think what is supported needs to be clear.